tests/test_wallet.py

```diff
@@ -96,6 +104,20 @@ async def test_mint_nft(self):
 
         assert tr
 
+    async def test_transfer_nft(self):
+        account_state = await self.wallet.get_account_state()
+        nfts = account_state.committed.nfts.values()
```

Comment:
Shouldn't this be `account_state.verified.nfts` since we have to wait until mint is verified to transfer?

Reply:
No. The nft might've been transfered already. The `minted_nfts` object should be used with verified